src/maintainer/adding_pkgs.rst

```diff
@@ -3,71 +3,94 @@
+     #. If it is an R package from `CRAN <https://cran.r-project.org/>`_, kindly
+     start by using the `conda-forge helper script for R recipes <https://github.com/bgruening/conda_r_skeleton_helper>`_ instead.
+     Then if necessary, you can make manual edits to the recipe.
+
+   .. tab:: Grayskull
+
+
+      Generate a python package recipe with Grayskull:
+      --------------------------------------------------
+      #. Install grayskull using conda through the conda-forge channel with the command: ``conda install -c conda-forge grayskull``
+      #. Fork and clone the `staged-recipes <https://github.com/conda-forge/staged-recipes>`_ repository from GitHub.
+      #. Checkout a new branch from the staged-recipes ``main`` branch.
+      #. Through CLI, cd inside the 'staged-recipes/recipes' directory.
+      #. Call grayskull and pass the pypi repository, followed by the name of the package you want to contribute to conda-forge. (i.e ``grayskull pypi <name-of-package>`` **OR** Use grayskull ``pypi <name-of-package> --strict-conda-forge`` to remove some selectors which are not necessary for conda-forge and adapt recipes to fit better in the conda-forge ecosystem.
+        Grayskull will create a folder with the same name as the package in the ‘recipes’ folder of the ‘staged-recipes’ directory. This folder will contain the meta.yaml file and also the license file if the package includes a license in the PyPI distribution.
+      #. Go through the generated ``meta.yaml`` file and remove all irrelevant comments in the ``meta.yaml``  file.. For simpler packages, the generated recipes are nearly perfect, but for some packages you might need to make certain tweaks.
+      #. Commit and push the changes. git add recipe/abc/meta.yaml git commit -m "add a commit message" git push
+      #. Create a PR.
+
+      Know more about `Grayskull <https://github.com/conda-incubator/grayskull>`_
+
+   .. tab:: Example recipe
+
+      Generate a recipe with example recipe
+      -------------------------
+      #. Source code should be downloadable, as a single file– as an archive (.tar.gz, .zip, .tar.bz2, .tar.xz) or tagged on GitHub,-- toensure that it can be verified. To know why see :ref:`tarballs_no_repos`
+      #. Fork and clone the `staged-recipes <https://github.com/conda-forge/staged-recipes>`_ repository from GitHub.
+      #. Checkout a new branch from the staged-recipes ``main`` branch.
+      #. Through CLI, cd inside the 'staged-recipes/recipes' directory.
+      #. Create a new folder within recipes, for your package (i.e., For unix: ``cp -r recipes/example recipes/<package_name>`` For windows: ``xcopy /h /e /i "recipes\example\" "recipes\<package_name>"``).
+      #. Copy `meta.yaml <https://github.com/conda-forge/staged-recipes/blob/master/recipes/
+        example/meta.yaml>`_ from the example directory. All the changes in the following steps will happen in the COPIED meta.yaml (i.e., `..staged-recipes/recipes/<name-of-package>/meta.yaml`).
```

Comment:
There's a strange spacing here in that URL. I don't know if it'll affect the output at all, but would look cleaner if the newline was removed.

Reply:
Also, that URL example after the i.e. should have two backticks, if you're trying to give it code formatting.